node/modules/lp2p/metrics.go

```diff
@@ -0,0 +1,30 @@
+package lp2p
+
+import (
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+)
+
+var otelmeter = otel.Meter("libp2p")
+
+var attrPeerID = attribute.Key("peer")
+var attrProtocolID = attribute.Key("protocol")
+var attrDirectionInbound = attribute.String("direction", "inbound")
+var attrDirectionOutbound = attribute.String("direction", "outbound")
+
+var otelmetrics = struct {
+	bandwidth metric.Int64ObservableGauge
+}{
+	bandwidth: must(otelmeter.Int64ObservableGauge("lotus_libp2p_bandwidth_total",
```

Comment:
It should have the suffix `bytes_total`.

Reply:
And it probably should be counter instead of gauge if we can guarantee monotonicity.